src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -187,6 +187,9 @@ class ov::genai::VLMPipeline::VLMPipelineImpl {
         SequenceGroup::Ptr sequence_group = std::make_shared<SequenceGroup>(request_id, prompt_ids, generation_config, block_size);
         requests.push_back(sequence_group);
 
+        OPENVINO_ASSERT((!m_is_chat_conversation || !std::get_if<std::function<StreamerRunningStatus(std::string)>>(&streamer)),
+                         "For chat mode, please, use Steamer as StreamerBase class or as callback with a bool return value.");
```

Comment:
Why is `StreamerRunningStatus` return value is prohibited for chat? You wrap it with `TextCallbackStreamer` below